src/gameObjects/map_start/chest.gameObject.js

```diff
@@ -0,0 +1,12 @@
+export const chest = (k, map, spawnpoints) => {
+    k.loadSprite('chest', './assets/sprites/chest.png');
+  
+    return k.add([
+      k.sprite('chest'),
+      k.pos(map.pos.x + spawnpoints.chest.x, map.pos.y + spawnpoints.chest.y),
```

Comment:
Unfortunately, there is no spawnpoint.chest available for now.
Have a look at other game object files to get an idea how you can place objects differently in such a case